test/Sentry.Tests/HubTests.cs

```diff
@@ -71,7 +71,7 @@ public void PushAndLockScope_DoesNotAffectOuterScope()
             hub.ConfigureScope(s => Assert.False(s.Locked));
         }
 
-        [Fact]
+        [Fact(Skip = "Flaky")]
```

Comment:
Best to raise issues to address these later, if there are no issues yet

Reply:
There are issues about specific tests, maybe one of them can be generalized.